vite.config.shared.ts

```diff
@@ -160,11 +161,21 @@ export const getApiProxies = (pathToRoot: string = '../../../.') => {
         ...konnectAuthHeader,
       },
     }
+    // @ts-ignore
+    regionalV2Proxies[`^/${region}/konnect-v2`] = {
```

Comment:
question: Is there a reason you moved away from the `kong-api/` proxied URL? We have this as a standard across most of the repos

Reply:
Previously, it was set to this:

`'/us/kong-api/konnect-api'`

It’s now being set to this:

`'/us/konnect-v2'`

Instead, you should use something like this to keep kong-api in the proxied URL:

`'/us/kong-api/v2/konnect-api'`

Reply:
@jillztom could u refer to the question?

Reply:
because I'm not sure about answer to that

Reply:
We do not have a region specific proxying for `v2` endpoints if I'm not mistaken. 

`'^/kong-api/v2'` is hitting the `https://global.api.konghq.tech`

Reply:
This is for the global one: https://github.com/Kong/public-ui-components/blob/5cf51320246505fba2958ea229aa9f7a71a5a5a5/vite.config.shared.ts#L187-L194

So then something like `'^/kong-api/us/v2' maybe? 

When Val and I set up all the proxying, we tried to standardize on `kong-api` in the path just to make it easy to debug, expectations, etc.

Reply:
We can do 

```
regionalV2Proxies[`^/${region}/kong-api/v2`] = {
      target: (env.VITE_KONNECT_API ?? '').replace(/\{geo\}/, region),
      rewrite: (path: string) => path.replace(`/${region}/kong-api`, ''),
      changeOrigin: true,
      headers: {
        ...konnectAuthHeader,
      },
    }
```

Just have to confirm it won't collide with `'^/kong-api/v2'`. It shouldn't because we are having this regional proxy before the global proxy.

Reply:
Correct, the region comes first so it won't conflict